waspc/src/Wasp/Generator/WebAppGenerator/Common.hs

```diff
@@ -97,3 +111,8 @@ mkUniversalTmplFdWithDst relSrcPath relDstPath =
     (webAppRootDirInProjectRootDir </> relDstPath)
     (universalTemplatesDirInTemplatesDir </> relSrcPath)
     Nothing
+
+toViteImportPath :: Path Posix (Rel r) (File f) -> Path Posix (Rel r) (File f)
+toViteImportPath = fromJust . SP.parseRelFileP . dropExtension . SP.fromRelFileP
+  where
+    dropExtension = fst . splitExtension
```

Comment:
This signature differs from the method `toESModulesImport` path we have in `ServerGenerator`.

I don't think it's a big deal, as we'll want to overhaul the entire story wrt these extensions in the near future. I can change the signature to `FilePath -> FilePath`, but then we get this: https://github.com/wasp-lang/wasp/pull/1090#discussion_r1157836550 (which is arguably even worse).